os/arch/arm/src/amebasmart/amebasmart_pmhelpers.c

```diff
@@ -115,8 +115,6 @@ int SOCPS_AONWakeReason(void)
 void pg_timer_int_handler(void *Data)
```

Comment:
Since the PM state change is moved to pm_wakehandler, I don't think we need this timer handler anymore. But you don't need to change it first,
I will do a cleanup once this PR is merged.